src/main/java/sirius/biz/tenants/ProfileController.java

```diff
@@ -106,6 +104,48 @@ public void profileChangePassword(WebContext ctx) {
         ctx.respondWith().template("/templates/biz/tenants/profile-change-password.html.pasta", userAccount);
     }
 
+    /**
+     * Invalidates all client sessions of the current user by changing the fingerprint.
+     *
+     * @param ctx the current request
+     */
+    @LoginRequired
+    @Routed("/profile/changeFingerprint")
+    public void profileChangeFingerprint(WebContext ctx) {
+        U userAccount = getUser().getUserObject(getUserClass());
+        userAccount = userAccount.getMapper().refreshOrFail(userAccount);
+        assertNotNew(userAccount);
+
+        userAccount.getUserAccountData().getLogin().resetFingerprint();
+        userAccount.getMapper().update(userAccount);
+
+        ctx.respondWith().redirectToGet("/profile");
```

Comment:
Wouldn't it be better to redirect the user to wondergem root in this case? I seems to me that this triggers a user being logged out instantly.